src/developer-quickstart.md

```diff
@@ -80,7 +80,7 @@ Here is the project that `Forc` has initialized:
 ```console
 $ tree fuel-project
 fuel-project
-└── counter_contract
+└── counter-contract
```

Comment:
Thank you so much for this! I actually have another PR here: https://github.com/FuelLabs/fuel-docs/pull/42/files to fix this. with `forc v0.26`, we don't get `tests/harness.rs` and `Cargo.toml` with `forc new` anymore. Maybe we should consolidate both PRs into one? If you incorporate my changes here, I can close that PR.

Reply:
awesome thanks @mohammadfawaz! Included the changes here: https://github.com/FuelLabs/fuel-docs/pull/43/commits/a17086d68bfb76328ffe80baca1a1b8c0f9c74c7